### ISSUE TITLE
builder machine seems to be out of inodes

### ISSUE THREAD

**Issue by Mic92** [nix-community/infra#1322]
I often see error messages like this in the build log of buildbot:

```
cannot link '/nix/store/.links/012j5w2vs5a83hn3s1iaqyjy986l7x3p3kpq3akx8nvvknd3h5np' to '/nix/store/y141zhwqzsz6lr8cby6lqw3ngyna8nlz-source/pkgs/applications/graphics/darktable/default.nix': No space left on device
```

My suspicion is that there are not enough inodes available. Also other builds sometimes randomly fail with out-of-disk errors sometimes.

Potentially solved by: https://github.com/nix-community/infra/pull/1207

**Comment by Mic92** [comment]
There is a chance that this error is non-critical.

**Comment by zowoq** [comment]
> My suspicion is that there are not enough inodes available.

> Potentially solved by: https://github.com/nix-community/infra/pull/1207

Yes, been meaning to fix this for a while, I should get it done sometime this week.

> Also other builds sometimes randomly fail with out-of-disk errors sometimes.

I haven't actually seen a build fail because of this, can you share a link?

**Comment by Mic92** [comment]
> > My suspicion is that there are not enough inodes available.
> 
> > Potentially solved by: #1207
> 
> Yes, been meaning to fix this for a while, I should get it done sometime this week.
> 
> > Also other builds sometimes randomly fail with out-of-disk errors sometimes.
> 
> I haven't actually seen a build fail because of this, can you share a link?

I had a linux kernel build going out of disp space, but now it works as well. I will link the build log, next time I see it.

**Comment by zowoq** [comment]
We've switched to zfs for this machine. https://github.com/nix-community/infra/pull/1207
